### PR TITLE
Merging code python3

### PR DESCRIPTION
**Pull request by Akshay14469**

### CODE REVIEW

**Review by usha13529 — DISMISSED** [review]
Changes looks good to me. Please do round of testing once changes deployed on paython3.

File: NHS.Api/Python2.Riak.Window/NHS_AnalyticsAPI/AnalyticConfig.py [hunk 1]
```diff
@@ -1,7 +1,16 @@
-config={
-        'Riak':{
-            'Ip':"35.176.37.177",'Port':8087,
-            'Buckets':{"Trust":"TrustPerformance","Region":"RegionPerformance","TRUST_REGION_MAP":"TrustRegionMap","Trusts_Region_Map":"test"},
-            'BucketsKey':{"Trust":"TrustData","Region":"RegionData","TRUST_REGION_MAP":"TrustRegionMapData"}
+config = {
+        'Riak': {
+            'Ip': "35.176.37.177", 'Port': 8087,
+            'Buckets': {
```
Comment (prabhat14176) [inline NHS.Api/Python2.Riak.Window/NHS_AnalyticsAPI/AnalyticConfig.py:4]: remove unwanted space

File: NHS.Api/Python2.Riak.Window/NHS_AnalyticsAPI/riak_connector.py [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 import riak
-from django.conf import settings
 from AnalyticConfig import config
 
-class Riak_Connector:
+
+class Riak_Connector(object):
     __rc=None
  
     def __init__(self):
```
Comment (prabhat14176) [inline NHS.Api/Python2.Riak.Window/NHS_AnalyticsAPI/riak_connector.py]: remove if there is any blank line

File: NHS.Api/Python2.Riak.Window/NHS_AnalyticsAPI/views.py [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
-    trust_list=[]
-    regions=rc.getRc().get_keys(config['Riak']['Buckets']['Trusts_Region_Map'])
+    trust_list = []
+    regions = rc.getRc().get_keys(config['Riak']['Buckets']['Trusts_Region_Map'])
     for region in regions:
-        trust_list.extend(rc.getRc().get_by_key(config['Riak']['Buckets']['Trusts_Region_Map'],region))
+        trust_list.extend(rc.getRc().get_by_key(config['Riak']['Buckets']['Trusts_Region_Map'], region))
     return JsonResponse(trust_list, safe=False)
 
+
```
Comment (prabhat14176) [inline NHS.Api/Python2.Riak.Window/NHS_AnalyticsAPI/views.py:18]: check if there is blank line

File: NHS.Api/Python3.Riak.Ubuntu/NHS_Analytics/settings.py [hunk 4]
```diff
@@ -12,9 +12,10 @@
     "rest_framework"
 ]
 
+# CORS ATTRIBUTES
+CORS_ORIGIN_ALLOW_ALL = True
 
-CORS_ORIGIN_ALLOW_ALL=True
-
+# APPLICATION ATTRIBUTES
```
Comment (prabhat14176) [inline NHS.Api/Python3.Riak.Ubuntu/NHS_Analytics/settings.py]: remove these settings from here , as they are moved to config file.

File: NHS.Api/Python3.Riak.Ubuntu/NHS_Analytics/settings.py [hunk 5]
```diff
@@ -24,8 +25,8 @@
 TRUST_REGION_MAP_BUCKET = "TrustRegionMap"
 TRUST_REGION_MAP_KEY = "TrustRegionMapData"
 
-# Application definition
 
+# Application definition
 INSTALLED_APPS = [
     'django.contrib.admin',
```
Comment (prabhat14176) [inline NHS.Api/Python3.Riak.Ubuntu/NHS_Analytics/settings.py:21]: check if CORS is added or not

**Review by prabhat14176 — COMMENTED** [review]
Looks good to me , once following changes are done

File: NHS.Api/Python2.Riak.Window/NHS_AnalyticsAPI/views.py (hunk 3 above)
Comment (Akshay14469) [reply NHS.Api/Python2.Riak.Window/NHS_AnalyticsAPI/views.py:18]: Its a space between 2 functions.

File: NHS.Api/Python3.Riak.Ubuntu/NHS_Analytics/settings.py (hunk 5 above)
Comment (Akshay14469) [reply NHS.Api/Python3.Riak.Ubuntu/NHS_Analytics/settings.py:21]: Its already added.

**Review by prabhat14176 — APPROVED** [review]
Approved

**Pull request merged**